cogs/modmail.py

```diff
@@ -174,11 +174,11 @@ def obj(arg):
         silent = str(message).lower() in {'silent', 'silently'}
         cancel = str(message).lower() == 'cancel'
 
-        if cancel and thread.close_task is not None and not thread.close_task.cancelled():
-            thread.close_task.cancel()
-            await ctx.send(embed=discord.Embed(color=discord.Color.red(), description='Scheduled close has been cancelled.'))
-            return
-        elif cancel:
+        if cancel:
+            if thread.close_task is not None:
```

Comment:
You don't need to check this since the function already checks it.

Reply:
I only reorganized this part. I think it's meant to check if the command is `close cancel`.

 - If there is a pending close task, close it. 
 - If not, output an error message.